src/main/scala/Stream.scala

```diff
@@ -0,0 +1,21 @@
+package icenet
+
+import chisel3._
+import chisel3.util._
+
+
+class StreamChannel(val w: Int) extends Bundle {
```

Comment:
Can you put a comment in this file saying that this bundle is duplicated in testchipip?